### ISSUE TITLE
Use correct routing depending to protocol.

### ISSUE THREAD

**Issue by maauso** [graphite-ng/carbon-relay-ng#135]
Anybody know if exist any way to add routes depending on the metric is tcp or pickle ?
I'm trading something like that

   ``` 'addRoute sendAllMatch default  10.2.5.215:2014 spool=true pickle=true ',```
    ``` 'addRoute sendAllMatch default  10.2.5.215:2003', ```

But the behavior that I have is that the metrics are sending to both ports.

**Comment by pawellesniewski** [comment]
Unfortunately, by default carbon-relay-ng sends metrics to all matching routes (i make some changes on my own to handle it - i mean to stop iteration on first match).

btw. pickle is the format of data - it is send over tcp also

You can try to start carbon-relay daemon on :2004 to accept pickle protocol and carbon-relay-ng on :2003 to accept single metrics.

**Comment by Dieterbe** [comment]
yes pickle=true is an output option. there is no method to do what you describe.
